private-templates-service/client/src/components/Designer/Designer.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { RootState } from '../../store/rootReducer';
+import requireAuthentication from '../../utils/requireAuthentication';
+import { connect } from 'react-redux';
+import { UserType } from '../../store/auth/types';
+
+//ACDesigner
+import * as monaco from 'monaco-editor';
+import markdownit from 'markdown-it';
+import * as ACDesigner from 'adaptivecards-designer';
+
+const mapStateToProps = (state: RootState) => {
+  return {
+    isAuthenticated: state.auth.isAuthenticated,
+    user: state.auth.user
+  };
+};
+
+interface DesignerProps {
+  isAuthenticated: boolean;
+  user?: UserType;
+  authButtonMethod: () => Promise<void>;
+}
+
+class Designer extends React.Component<DesignerProps>{
+
+  private designer: ACDesigner.CardDesigner | undefined;
```

Comment:
`private designer?: ACDesigner.CardDesigner;`

Reply:
oh i realised that might not work

Reply:
it works